dev_guide/builds.adoc

```diff
@@ -1362,9 +1366,18 @@ git:
     name: "<committer name>"
     email: "<committer e-mail>"
   message: "<commit message>"
+env:
+   - name: "<variable name>"
+     value: "<variable value>"
+
 ----
 ====
 
+Similar to the link:#buildconfig-environment[*BuildConfig* environment] variables, the environment variables
+defined in the above payload will be made available to your build.  When variable names in the above payload
+collide with the names of the link:#buildconfig-environment[*BuildConfig* environment] variables, the variables
+in the payload will take precedence.
```

Comment:
(I know you have it in the footnote, but i don't like the idea of the footnote being the only documentation of the behavior)